truffle/src/main/java/org/jruby/truffle/RubyFiletypeDetector.java

```diff
@@ -0,0 +1,50 @@
+package org.jruby.truffle;
```

Comment:
The FiletypeDetector isn't Truffle-only, it might be useful for whole JRuby - so consider option of putting it to generic JRuby runtime.